src/scss/partials/_body.scss

```diff
@@ -0,0 +1,14 @@
+// $background-color-mix: #fdfcfb;
+// $background-color-mix-with: #e2d1c3;
+
+body {
+  font-family: $font-family;
+  background: $primary-color;
+}
+
+footer {
+  margin: 0 auto;
+  width: 100%;
+  max-width: 250px;
```

Comment:
If you use the row/col system correctly here you won't need max-width

Reply:
yes i added div and class col in my all 3 pages with width class and padding and margin class